openbb_terminal/parent_classes.py

```diff
@@ -134,9 +134,10 @@ def __init__(self, queue: List[str] = None) -> None:
         self.completer: Union[None, NestedCompleter] = None
 
         self.parser = argparse.ArgumentParser(
-            add_help=False, prog=self.path[-1] if self.PATH != "/" else "terminal"
+            add_help=False,
+            prog=self.path[-1] if self.PATH != "/" else "terminal",
+            exit_on_error=False,
```

Comment:
Looks like inside our MacOS CI : `ArgumentParser` doesn't have `exit_on_error`.

A solution for the tests to pass would be to remove this argument from `ArgumentParser`:
```python
exit_on_error=False,
```